README.rst

```diff
@@ -272,3 +272,12 @@ As in the case of abbreviations, you can hear the audio for a given sentence and
 .. _tab separated file: https://en.wikipedia.org/wiki/Tab-separated_values
 .. _common.py: https://github.com/mozilla/CorporaCreator/blob/master/src/corporacreator/preprocessors/common.py
 .. _en.py: https://github.com/mozilla/CorporaCreator/blob/master/src/corporacreator/preprocessors/en.py
+
+
+Development
+===========
+
+::
+
+    $ pip install -r requirements.txt
+    $ pytest
```

Comment:
Testing is also not done this way. (See for example [here](https://github.com/mozilla/CorporaCreator/blob/master/setup.cfg#L63).) So this should be removed.

Reply:
:+1: